libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/client/EvcsClient.java

```diff
@@ -197,24 +199,23 @@ private void checkResponseStatusCode(HttpResponse<String> evcsHttpResponse)
             } catch (JsonProcessingException e) {
                 responseMessage = "Failed to parse evcs response body.";
             }
-            LOGGER.info(
+
+            LOGGER.error(
                     LogHelper.buildLogMessage(
                                     ErrorResponse.RECEIVED_NON_200_RESPONSE_STATUS_CODE
                                             .getMessage())
                             .with(LOG_STATUS_CODE.getFieldName(), statusCode)
-                            .with(LOG_MESSAGE_DESCRIPTION.getFieldName(), responseMessage));
+                            .with(LOG_RESPONSE_MESSAGE.getFieldName(), responseMessage));
```

Comment:
We shouldn't set `LOG_MESSAGE_DESCRIPTION` - this will overwrite the message above